src/layer/AbstractLayer.ts

```diff
@@ -3,6 +3,7 @@ import axios from 'axios';
 import { GetMapParams, ApiType, PaginatedTiles } from 'src/layer/const';
 import { BBox } from 'src/bbox';
 import { Dataset } from 'src/layer/dataset';
+import { IRequestConfig } from 'src/utils/axiosInterceptors';
```

Comment:
Nitpick: since all of the types (which is how we are using `IRequestConfig`, though it's an interface) are named without the leading `I`, can we drop it here too for consistency? (just `RequestConfig`)

Reply:
I originally had an issue with this, and decided to leave it as is. I have now realised there were a duplicate in `eslintrc.js`, so the issue is now resolved